server/a_socials/settings/base.py

```diff
@@ -197,6 +197,7 @@
 
 CORS_ALLOW_HEADERS = default_headers + (
   'Authorization',
+  'Token'
```

Comment:
I added this because I was having CORS issues while trying to interact with the graphql endpoints from the react application. This is to allow the token to be passed in the headers with a key word Token

Reply:
@OsayamenOsaretin you might want to take a look at this to determine it's validity

Reply:
@andela-sjames, I have confirmed that this is indeed needed. Our authorization procedure does need the `Token` header.
We used the `Authorization` header for the default JWT implementation.